platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -501,58 +547,75 @@ check_shared_library_dependency() {
     # IOTEDGE_COMMON_SHARED_LIBRARIES is for both aziot-edged and aziot-identityd
```

Comment:
can you remove this comment? as var name changed

Reply:
Done